### PR TITLE
Add howto release

### PR DESCRIPTION
**Pull request by atedstone**
These instructions are based on OGGM (https://github.com/OGGM/oggm/blob/master/HOWTO_RELEASE).

Broadly, part of #34

### CODE REVIEW

File: HOWTO_RELEASE [hunk 1]
```diff
@@ -0,0 +1,45 @@
+How to issue a GeoUtils release.
+
+ 1. Go to your local main repository (not the fork) and ensure your master branch is synced:
+       git checkout master
+       git pull
+ 2. Look over whats-new.rst and the docs. Make sure "What's New" is complete
+    (check the date!) and add a brief summary note describing the release at the
+    top.
+ 3. If you have any doubts, run the full test suite one final time!
+      pytest --run-slow --mpl .
```
Comment (adehecq) [inline HOWTO_RELEASE:10]: Can't get this to run on my machine. Does it require a specific pytest version?

File: HOWTO_RELEASE [hunk 2]
```diff
@@ -0,0 +1,45 @@
+How to issue a GeoUtils release.
+
+ 1. Go to your local main repository (not the fork) and ensure your master branch is synced:
```
Comment (adehecq) [inline HOWTO_RELEASE:3]: > not the fork
Does this mean one need to clone the Glaciohack repo?

**Comment by adehecq** [comment]
Made some minor comments, apart from that it seems good!

**Review by adehecq — APPROVED** [review]

File: HOWTO_RELEASE (hunk 2 above)
Comment (atedstone) [reply HOWTO_RELEASE:3]: yes.

File: HOWTO_RELEASE (hunk 1 above)
Comment (atedstone) [reply HOWTO_RELEASE:10]: me neither. Can't find out anything useful by googling. @fmaussion , any tips?

Comment (fmaussion) [reply HOWTO_RELEASE:10]: `--mpl` is for matplotlib image tests, you probably don't need that.

File: HOWTO_RELEASE (hunk 2 above)
Comment (fmaussion) [reply HOWTO_RELEASE:3]: I recommend to have only one or two persons in charge of doing releases - this needs a bit coordination.

**Pull request merged**